File: java/Java入門2/problem_013.java
// おみくじを作る

public class Main {
	public static void main(String[] args) {
		int omikuji = (int)(Math.random() * 100 + 1);  // randomメソッドについては次のチャプターで説明します
		if (omikuji >= 30) {
			System.out.println("omikujiの中身は" + omikuji + "なので大吉");
		} else {
			System.out.println("omikujiの中身は" + omikuji + "なので大凶");
		}
	}
}

File: Java Script/Java Script入門/problem_013.js
// 数の表示とサイコロ(演習１)
process.stdin.resume();
process.stdin.setEncoding('utf8');
var number = parseInt(Math.random() * 6) + 1;
console.log("サイコロの目は"　+ number + "です");  //number変数に6の中から1つ（*６）、1から(+1)
